### ISSUE TITLE
[Desktop UI] Create empty dashboard view

### ISSUE THREAD

**Issue by apriofrost** [chef/automate#3335]
## User Story
After the Desktop Management offerings are available to the IT managers, but no Chef Infra Client on the desktops/laptops have been properly configured to report back to Automate, they logged in to Automate, select Desktops from the global navigation. They see an empty view of the dashboard.

The same view applies at individual cards when specific reports failed to sent to Automate.

**Special case**: The "Top 10 Errors" card has a case when the data is sent to Automat successfully but there is no error reported across the fleet

### Large card
![image](https://user-images.githubusercontent.com/9401459/82970913-4af18e00-9f86-11ea-9efc-98c74ce77f26.png)

Design Spec: https://chef.invisionapp.com/d/main#/console/19726898/412780943/inspect

### Small card

<img src="https://user-images.githubusercontent.com/9401459/82971238-13cfac80-9f87-11ea-89f1-3661178256f4.png" width="280">

Design Spec: https://chef.invisionapp.com/d/main#/console/19726898/413161891/inspect 
(The current global nav is non-responsive, will be addressed in a separate issue 😄 )

### Special case for "Top 10 Errors" card
Using the native emoji of the user's OS of `Party Popper` ([learn more in Emojipedia](https://emojipedia.org/party-popper/))

![image](https://user-images.githubusercontent.com/9401459/82971076-b176ac00-9f86-11ea-8a00-3002db4b66b9.png)
Design Spec: https://chef.invisionapp.com/d/main#/console/19726898/418724175/inspect

## URL change
Add a top-level item after the chef automate URL like `{chef-automate-url}/desktop`

## Definition of Done
Display the dashboard empty view as specified in the mockup, using Carbon UI components with customized visual styles. 

## Demo Script

1. Enable the Desktop feature in Automate without sending related data to Automate
2. Click the Desktop item from the global nav and show the empty page
3. Enable some but not all of the data to Automate, show how individual cards can be displayed in an empty state.

**Comment by susanev** [comment]
hmm, so its only one action to take to get data on all 4 of these cards, do i understand that correct? do users get value out of displaying it as 4 cards in this empty state? 

additionally, could we please get a component card added to the [design system repo](https://github.com/chef/chef-design-system).

also, has it been decided that desktop is a top-level nav item? id really like to advocate for it being part of dashboards if at all possible. content will be coming soon after and its a cross-world piece and needs to be a top-nav item and there won't be room for it. prolly a good convo to have with the whole ux team sometime soon.

**Comment by apriofrost** [comment]
@susanev there are multiple actions needed to get data on all four of these cards, docs should be created to explain that. As the first stab, we are unlikely to have the docs ready to link from the empty state view. We will keep iterating on the empty view design to add more values to users in the future.
The design pattern I'm following is from here: https://www.carbondesignsystem.com/patterns/empty-states-pattern/
I'm fine with adding the headings and subheadings back though, that's actually how I designed it the first design. I changed to in order to follow the Carbon design guideline.

The reason for desktops being a top-level nav item:
- it is a new offering we are promoting to users, it is a different group of customers from our existing one.
- the page layout is so different from anything under "Dashboard", using a new top-level nav item makes more sense to me
- I agree with your concerns about having too much in the top nav. I think the right thing for us to do next is to switch to the global left-side nav we've been talking about and it will also address the non-responsive issue. Happy to chat with the UX team a team about this. :)
